src/Point.cpp

```diff
@@ -43,7 +43,7 @@ Point::Point() : interpolation(BEZIER), handle_type(AUTO)
 	Initialize_Handles();
 }
 
-// Constructor which creates a single coordinate at X=0
+// Constructor which creates a single coordinate at X=1
```

Comment:
Like, for this one, you only need to look down 3 lines to see that it creates a `Coordinate(1, y)`. There's no reason for the value to be in the comment at all, IMHO.